internal/person.go

```diff
@@ -15,7 +15,7 @@ func (p *Person) Matches(filters Filters) (bool, error) {
 		if !ok {
```

Comment:
Would it be nicer to group all failing attributes for error message or just fail at first one?